src/bootslide.js

```diff
@@ -72,17 +72,20 @@ Bootslide.prototype.reset = function () {
 }
 
 Bootslide.prototype.resetMargins = function (ctx) {
+  var self = this
   $('.bootslide-section', ctx).css('transform', function (i) {
     // Figure out width of all previous sections (ie sections to the left)
     var prevWidth = $(this).prevAll()
       .map(function () { return $(this).width() }).get()
       .reduce(function (sum, current) { return sum + current }, 0)
 
+    var translateValue = self.isRtl() ? prevWidth * -1 : prevWidth
+
     // Store the point that bootslide-menu-slider has to translate to
     // to show this section
-    $(this).data('target-translate', prevWidth * -1)
+    $(this).data('target-translate', translateValue)
 
-    return 'translate(' + prevWidth + 'px,0)'
+    return 'translate(' + translateValue + 'px,0)'
```

Comment:
Previously the value here and the value two lines up were not the same. Are you sure it’s correct for them to now be the same?

Reply:
I think this was functioning similarly since I added the `* -1` to the `.slide` function, but I changed this back now.

Reply:
This is being a little nit-picky, so I apologize if it's annoying. But we're trying to now use modern JS when updating files. So when creating new variables, let's roll w/ either `const` or `let` depending what's more appropriate. `var` is leaky.

Reply:
I went ahead and made this change